src/mmw/js/src/core/modals/views.js

```diff
@@ -388,8 +389,34 @@ var PlotView = ModalBaseView.extend({
     }
 });
 
+var CreateAccountPersuasionModalView = ModalBaseView.extend({
+    template: modalCreateAccountPersuasionTmpl,
+
+    ui: {
+        'login': '[data-action="login"]',
+        'signup': '[data-action="signup"]',
+    },
+
+    events: _.defaults({
+        'click @ui.signup': 'signUp',
+        'click @ui.login': 'login'
+    }, ModalBaseView.prototype.events),
+
+    signUp: function() {
+        this.options.app.getUserOrShowSignUp(this.options.onSuccess);
+        this.$el.modal('hide');
+    },
+
+    login: function() {
+        this.options.app.getUserOrShowLogin(this.options.onSuccess);
+        this.$el.modal('hide');
```

Comment:
These two methods have the same implementation -- could they just be made into a single method?